vm/devices/pci/pci_core/src/spec.rs

```diff
@@ -38,7 +38,7 @@ pub mod hwid {
     }
 
     open_enum::open_enum! {
-        /// ClassCode identifies the PCI device's type.
+        /// ClassCode identifies the PCI device's type, the "Base Class".
```

Comment:
Note for reviewers: these changes are to match the terminology in [PCI Code and ID Assignment Specification](https://pcisig.com/sites/default/files/files/PCI_Code-ID_r_1_12__v9_Jan_2020.pdf) from the PCI SIG standards body.